src/implementations/HtmlEditorApiClientImpl.ts

```diff
@@ -88,11 +90,13 @@ export class HtmlEditorApiClientImpl implements HtmlEditorApiClient {
       content.type === "html"
         ? {
             htmlContent: content.htmlContent,
+            previewImage: content.previewImage,
             type: "html",
           }
         : {
             meta: content.design,
             htmlContent: content.htmlContent,
+            previewImage: content.previewImage,
```

Comment:
@Cromeror @maurocardinali what do you think about the name `previewImage` for the field in the API payload?

Reply:
I like thumbnail if just is a miniature picture.

Reply:
So, preview image is ok!! 🚀